main.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<title>marketkurly</title>
+	<link rel="stylesheet" href="reset.css" />
+	<link rel="stylesheet" href="common.css" />
+	<link rel="stylesheet" href="main.css" />
+</head>
+<body>
+	<header class="header">
+		<div class="header-content">
+			<div class="account-box">
+				<a class="account">회원가입</a>

```

Comment:
a tag인데 href 가 없습니다.
```suggestion
				<a class="account" href="#">회원가입</a>

```

Reply:
반영했습니다.